README.md

```diff
@@ -4,10 +4,16 @@
 </p>
 
 <h3 align="center">
-  Lightning-Fast Frontend Platform for Headless Commerce
+  <strong>An ecosystem of developer tools that accelerate and simplify building eCommerce Storefronts</strong>
 </h3>
 <p align="center">
-    Vue Storefront is the most popular and most advanced Frontend Platform for eCommerce.
+    Works with any backend
```

Comment:
The team has discussed the wording of "_any_ backend" in the past and have avoided in the past in favor of being more realistic about the primary use case being our pre-built integrations.